.travis/environments/travis/private.yml

```diff
@@ -34,4 +34,3 @@ localsettings_private:
   SECRET_KEY: "JHX1y7iG0bQaVICcBXrQYw1fcJcomC24p6lxAfKMG0B5g7D5"
   TOUCHFORMS_API_PASSWORD: 'TsyasEivKf4GtbQX'
   TOUCHFORMS_API_USER: 'touchforms@example.com'
-
```

Comment:
(Minor) Needs a newline at end of file

Reply:
I think this previously ended with _two_ newlines and I deleted one of them. (Github will put a "No newline at end of file" message if there's no newline.)